go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/openshift/aws-account-shredder
 go 1.13
 
 replace (
-	bitbucket.org/ww/goautoneg => github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822
+	bitbucket.org/ww/goautoneg => github.com/munnerz/goautoneg v0.0.0-20190414153302-2ae31c8b6b30
```

Comment:
I couldn't get things running without this. Is it an issue on my end alone? If so I will remove the change